src/sonic-config-engine/minigraph.py

```diff
@@ -466,6 +483,7 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['ntp_servers'] = ntp_servers
     results['forced_mgmt_routes'] = mgmt_routes
     results['erspan_dst'] = erspan_dst
+    results['deploymentId'] = deploymentId
 
     return results
 
```

Comment:
Could you add a sample minigraph with bgp peers and corresponding test to the test folder?